src/news-update/index.md

```diff
@@ -0,0 +1,21 @@
+# What is a news update?
```

Comment:
I rewrote this based on below comments:

# What is a news update?

News updates are a great way to garner your game some new or renewed publicity. They are the primary means of staying in contact with your game's followers. Post news updates whenever something interesting happens with your game. Followers will be sent notifications and the news will be archived on your game page. Your news updates might even show up on the Game Jolt homepage.

The best news updates deliver information that a game's followers would want to know. If you upload a new version of your game, hit a major milestone on development, have updated changelogs or teasers, you should take the time to make a news update.

If you have something to post that is not a news update about a particular game, you can use your developer blog.

## How do I post a news update?

Best news updates are occasional and contain important information with supporting media files. 

The content is written in Markdown, which is easy to use and allows you to do a number of cool things without having to use HTML. For help with Markdown, consult our [Markdown guide](https://help.gamejolt.com/markdown).

You can upload media files such as screenshots and game art through [Markdown](https://help.gamejolt.com/markdown). Although video is currently not an option via Markdown, you can always add animated gifs!

Be sure to space out your news updates and limit yourself to at most 1 update per week. If you post too often, your followers may ignore your update and even unfollow your game. If your news is too short for a news update, utilize [Twitter](www.twitter.com)!

To create a news update for a particular game, go to your [developer dashboard](http://gamejolt.com/dashboard/) and select the game from under "Your Games". On the game's overview page, visit the "News" tab and click "Add Game News".